app/models/payment_confirmation.rb

```diff
@@ -0,0 +1,87 @@
+class PaymentConfirmation
```

Comment:
Yeah, I considered it a couple of times, but discounted it because, although it seems on the surface, this is an Active(Record|Model)-like thing, the validations we're doing are across whole spectrum of stuff, rather than against individual fields as such, so the metaphor doesn't _quite_ carry

Reply:
Makes sense 👍